examples/hikari_lightbulb_basic_queue/music_plugin.py

```diff
@@ -0,0 +1,255 @@
+import os
+import logging
+from posix import environ
+
+import hikari
+import lightbulb
+import lavasnek_rs
+
+PREFIX = "!" # Should probably set this to an environment variable
+TOKEN = os.environ["DISCORD_TOKEN"]
```

Comment:
Could you import these constants from `main.py`?

Reply:
Done

Reply:
You didn't do this.